modules/openy_gc_auth/src/GCUserAuthorizer.php

```diff
@@ -72,11 +85,23 @@ public function authorizeUser($name, $email, array $extra_data = []) {
         $account->save();
       }
     }
+    // List of roles to redirect user login page.
+    $userRolesArray = [
+      'administrator',
+      'virtual_ymca_editor',
+    ];
+    // Redirecting user login page.
+    foreach ($userRolesArray as $role) {
+      if ($account->hasRole($role)) {
+        $loginUrl = Url::fromRoute('user.login')->toString();
+        $this->messenger->addMessage($this->t('Please retry your login on this form.'));
```

Comment:
This line is throwing an error:
> Error: Call to undefined method Drupal\openy_gc_auth\GCUserAuthorizer::t() in Drupal\openy_gc_auth\GCUserAuthorizer->authorizeUser() (line 97 of /.../docroot/modules/contrib/openy_gated_content/modules/openy_gc_auth/src/GCUserAuthorizer.php) 

We're reviewing for a fix.

Reply:
https://drupal.stackexchange.com/questions/257631/call-to-undefined-function-t-in-an-unit-test has some possible solutions to this.

Reply:
Updated the code as per suggestions, Please review.